pkg/auth/providers/ldap/ldap_provider.go

```diff
@@ -32,8 +32,16 @@ const (
 	OKTAName       = "okta"
 )
 
+// An ErrorNotConfigured indicates that the requested LDAP operation
+// failed due to missing or incomplete configuration.
+type ErrorNotConfigured struct{}
+
+// Provides a string representation of an ErrorNotConfigured
+func (e ErrorNotConfigured) Error() string {
```

Comment:
```suggestion
// Error provides a string representation of an ErrorNotConfigured
func (e ErrorNotConfigured) Error() string {
```
Comments should start with the name of the function.